src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link nav-list-link"><img src="./images/logo.png" alt="MoyoLogo"></a>
+      <nav class="nav-block">
+        <ul class="nav-list">
```

Comment:
[Rules](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
Add empty lines between multiline sibling blocks of HTML
Please, fix it in all cases